src/config.h

```diff
@@ -25,9 +25,12 @@ const int MAX_LAYER_CNT = 256;
 struct LayerConfig : public ConfigUtils::Config {
     APPCFG_CLASS;
 
+    APPCFG_PARA(gzipIndex, std::string, "");
     APPCFG_PARA(file, std::string, "");
+    APPCFG_PARA(dataFile, std::string, "");
```

Comment:
dataFile and dataDigest => targetFile and targetDigest?
Keep consistent with upper